setup.py

```diff
@@ -10,7 +10,8 @@
 with open('HISTORY.rst') as history_file:
     history = history_file.read()
 
-requirements = ['snakebids==0.2.1' ]
+requirements = ['snakebids @ git+https://github.com/akhanf/snakebids.git@suffix_wildcard',
```

Comment:
Maybe we can release Snakebids 0.3.1 once the suffix-as-wildcard change gets in, then change this to `snakebids>=0.3.1`.

Reply:
sounds good, just had this as a placeholder for now

Reply:
Okay, I think we can change this to `snakebids==0.3.0` now and merge the PR.

Reply:
yep, I'll make that change now